src/main/java/com/three_tech_solutions/slot_app/data/mappers/StudentMapper.java

```diff
@@ -32,7 +34,7 @@ public Student toStudent(CreateStudentRequest studentDTO, Plan plan, User user)
     }
 
 
-    public StudentResponse toStudentResponse(Student student) {
+    public static StudentResponse toStudentResponse(Student student) {
```

Comment:
Aca no hace falta que el método sea `static` porque el mapper es un bean de Spring (porque esta denotado como un `@Service`). Entonces en vez de acceder mediante la clase estatica deberiamos hacerlo mediante la relacion entre la clase Service y el Mapper

```suggestion
    public StudentResponse toStudentResponse(Student student) {
```